inf-clojure.el

```diff
@@ -238,11 +238,17 @@ This should usually be a combination of `inf-clojure-prompt' and
 `inf-clojure-subprompt'."
   :type 'regexp)
 
-(defcustom inf-clojure-prompt-on-set-ns t
+(defcustom inf-clojure-prompt-on-commands t
```

Comment:
the new name would require to flip the logic and rename also the other functions...also maybe in the future not all the commands will take symbol at point so I was wondering if we could leave it like this.